addons/github/requirements.txt

```diff
@@ -1,4 +1,4 @@
 cachecontrol==0.10.2
-github3.py==0.9.0
+github3.py==1.0.0a3
```

Comment:
Latest version is `1.0.0a4`. Any reason not to pin to that one? See [changelog](http://github3py.readthedocs.io/en/latest/project_changelog.html)

Reply:
I will go ahead and bump it to `1.0.0a4`. Initially, the goal was to bump up enough versions to solve the issue.